### PR TITLE
ea documentation

### PR DESCRIPTION
**Pull request by stovak**

### CODE REVIEW

**Review by jspellman814 — APPROVED** [review]
LGTM

File: docs/Customer Secrets EA.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+For multiple private repositories, you will need to create a COMPOSER_AUTH json and make it available via the COMPOSER_AUTH environment variable.
+
+Composer has the ability to read private repository access information from the environment variable: COMPOSER_AUTH. The COMPOSER_AUTH variables has to be in a [specific JSON format](https://doc.codingdict.com/composer/doc/articles/http-basic-authentication.html). 
+
+That format example is here:
+
+```bash
+read -e COMPOSER_AUTH_JSON <<< {
+    "http-basic": {
+        "github-oauth.github.com": {
```
Comment (greg-1-anderson) [inline docs/Customer Secrets EA.md:57]: I think that `github-oauth.github.com` is out of place here.

File: docs/Customer Secrets EA.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+1. `export SITE_NAME=$USER-secrets-testing`
+
+1. `terminus site:create ${SITE_NAME} “${USER} Secrets Testing”  ${SITE_UPSTREAM_ID}`
+
+1.  `terminus local:clone ${SITE_NAME}`
+
+1. Set secrets using one of the methods shown below. Changing a file and pushing the code to development will trigger a composer run.
+
+***Github token authentication***
+
```
Comment (greg-1-anderson) [inline docs/Customer Secrets EA.md:38]: We could use a sentence here about Github oauth, perhaps with a link to https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token

**Review by greg-1-anderson — CHANGES_REQUESTED** [review]
c.f. other comments: intro github oauth, remove github-oauth.github.com example from http basic auth example

File: docs/Customer Secrets EA.md (hunk 1 above)
Comment (stovak) [reply docs/Customer Secrets EA.md:57]: you think it should be GitHub.com ?

File: README.md [hunk 3]
```diff
@@ -93,3 +93,85 @@ Note that prior to running the tests, you should first run:
...
+
+***To create a fresh site for testing***
+
+1. Set the upstream to the ID of our Drupal 9 composer-based upstream: 
+
+	 `export SITE_UPSTREAM_ID=bde48795-b16d-443f-af01-8b1790caa1af`
+
+1. `export SITE_NAME=$USER-secrets-testing`
+
+1. `terminus site:create ${SITE_NAME} “${USER} Secrets Testing”  ${SITE_UPSTREAM_ID}`
```
Comment (greg-1-anderson) [inline README.md:127]: `drupal-composer-managed` works in place of `${SITE_UPSTREAM_ID}`

**Review by greg-1-anderson — APPROVED** [review]
LGTM. Made one optional suggestion to simplify site:create example.

**Review by jspellman814 — APPROVED** [review]

**Pull request merged**